app/models/base_organisation.rb

```diff
@@ -3,7 +3,17 @@ class BaseOrganisation < ActiveRecord::Base
   validates_url :website, :prefferred_scheme => 'http://', :if => Proc.new{|org| org.website.present?}
   validates_url :donation_info, :prefferred_scheme => 'http://', :if => Proc.new{|org| org.donation_info.present?}
   has_many :category_organisations, :foreign_key => :organisation_id
-  has_many :categories, :through => :category_organisations, :foreign_key => :organisation_id
+  has_many :categories, :through => :category_organisations, :foreign_key => :organisation_id do
+    def what_they_do
+      where(charity_commission_id: 100..199).order(name: :asc)
+    end
+    def who_they_help
+      where(charity_commission_id: 200..299).order(name: :asc)
+    end
+    def how_they_help
+      where(charity_commission_id: 300..399).order(name: :asc)
+    end
+  end
```

Comment:
they are @johnnymo87 and I had a little try and referring to them, but couldn't immediately see how to DRY these out - it would be great to be able to refer to these scopes cross-model - have the category stuff only defined in one place.

Although I just found this which makes for interesting reading:

https://gorails.com/blog/activerecord-merge

which implies we could do something like:

```
def what_they_do
  merge(Categories.what_they_do)
end
```

I just tried that from the command line and it works, and seems like we could go further and use `method_missing` or delegation to just directly pass any of these directly to the category class, although I guess what we'd really like for maximum flexibility is just to be adding all Category scopes to the set of available scopes so that this method could be used in combination with any Organisation or Category scope